### ISSUE TITLE
Odriod N2+ Does not Boot from eMMC

### ISSUE THREAD

**Issue by keypunch416** [MichaIng/DietPi#5639]
#### Required Information
- DietPi version | `8.6.1`
- Distro version | `echo $G_DISTRO_NAME $G_RASPBIAN`
- Kernel version | unknown as will not boot
- SBC model | Odroid N2+
- Power supply used | ALLPOWER 15VDC/4A
- eMMC card used | Label on eMMC antistatic bag "eMMC Memory Board - 16G", number below bar code "114991509", "SEED STUDIO", "M0A200110009", "01/18/20"

#### Additional Information (if applicable)
- Software title | DietPi_OdroidN2-ARMv8-Bullseye.img
- Was the software title installed freshly or updated/migrated?DietPi_OdroidN2-ARMv8-Bullseye.img  image freshly imaged to new never used eMMC prior (i.e. out just out anti static package)
- Can this issue be replicated on a fresh installation of DietPi? Yes

- Bug report ID | N/A

#### Steps to reproduce

1. 7z x /mnt/nfs/Data/Downloads/DietPi/DietPi_OdroidN2-ARMv8-Bullseye.7z
2. USB to eMMC adapter (label on anti static package "Pine64", "USB ADAPTER FOR EMMCMODULE", number below bar code "8 50796 00710 9", "SKU:USB-EMMC", silk screen on rear of PCB ""USB Progame to EMMC of Pine64", "20180410")
3. sudo dd if=/mnt/nfs/Data/Downloads/DietPi/DietPi_OdroidN2-ARMv8-Bullseye.img of=/dev/sdf bs=2048k oflag=dsync ; time sudo sync via USB to eMMC adapter known to work with same eMMC modules (I have 5 of them ordered same time for order received 20 July 2022)
4. Connected eMMC to Odriod board no SD card inserted and switch on eMMC position and not SPI position

#### Expected behaviour

- Expected DietPi would boot to login prompt as has done with Manjaro-ARM on another one of same (5) eMMCs ordered same time imaged using same USB to eMMC adapter (label on anti static package "Pine64", "USB ADAPTER FOR EMMCMODULE", number below bar code "8  50796 00710  9", "SKU:USB-EMMC", silk screen on rear of PCB ""USB Progame to EMMC of Pine64", "20180410")

#### Actual behaviour

- All sorts of output pause and repeats what looks like attempts to boot from the eMMC unsuccessfully

#### Extra details

- Cannot attach any console or logs as occurs on HDMI display
-  eMMC SanDisk 501N8BD04 16G (difficult to read as weak lettering to start likely rubbed off due to finger tip required to secure eMMC to USB adapter and to Odriod N2+
- Information on chip was as best could read and unable to read another line of letters/numbers on chip
- Same DietPi image file imaged to a 32GB Kingston Canvas MicroSD card boots to login prompt and functions like any Linux based OS

**Comment by Joulinar** [comment]
I guess it is the same like this https://dietpi.com/forum/t/odroid-n2-boot-up-problem-with-emmc/13359

**Comment by keypunch416** [comment]
No not same issue.  As noted in the open of this issue there screen is far from blank and ongoing retry activity for as long as power is connected all be it likewise not successful.

**Comment by keypunch416** [comment]
As I am sure many have experienced while searching wit completely different search terms for completely different area of interest in search I happened upon this that seems would be helpful and indicating I am not only one:

<https://forum.armbian.com/topic/20206-odroid-n2-issues-with-recent-firmware-and-emmc-modules/>

In my case I have the red eMMC colored modules and as the above link user noted no issues using these modules with Manjaro-ARM

As side point the search I was and have been working on for past 2 days is how to enableN2+  SPI with Diet Pi.  I gave up using Manjaro-ARM after day as not able to find how to enable SPI with Manjaro-ARM.  This should not be rocket science.  Be clear this issue is about DietPi not booting from eMMC and not about my SPI challenges.  I am simply noting my search for how to enable SPI on DietPi turned up the above link that did not turn up when I was searching for solutions for inability to boot DietPi from eMMC before I opened this issue.
